tests/store/test_store_api.py

```diff
@@ -1,13 +1,16 @@
 import datetime
-from typing import List, Optional
+import subprocess
+from typing import Dict, List, Optional
 
 import pytest
 
 from tests.apps.tower.conftest import CaseName
 from tests.mocks.store_mock import MockStore
-from trailblazer.constants import TrailblazerStatus
+from trailblazer.constants import CharacterFormat, TrailblazerStatus
 from trailblazer.store.models import Analysis
 
+FUNC_GET_SLURM_SQUEUE_OUTPUT_PATH: str = "trailblazer.store.crud.update.get_slurm_squeue_output"
```

Comment:
Should this constant be defined only once in a constant file?

Reply:
It is only for particular tests and I think it makes sense to define it where it is mocked as it has a very special use.